core/templates/domain/state/state-version-history.model.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Model class for creating domain object for exploration state
+ * version history.
+ */
+
+export interface StateVersionHistoryBackendDict {
+  'previously_edited_on_version': number;
```

Comment:
Let's make the names here also consistent with the backend names.

Reply:
Done.